app/auth-portal/src/pages/RumReport.vue

```diff
@@ -0,0 +1,119 @@
+            tone="action"
+            variant="solid"
+            @click="loadRumReport()"
+          >
+            Load Report
+          </VButton>
+        </Stack>
+
+        <template v-if="rumReportReqStatus.isPending">
+          <Icon name="loader" />
+        </template>
+        <template v-else-if="rumReportReqStatus.isError">
+          <ErrorMessage :requestStatus="rumReportReqStatus" />
+        </template>
+        <template v-else-if="rumReportReqStatus.isSuccess">
+          <div class="relative mt-4">
+            <div class="text-lg font-bold mb-2">
+              Total Entries: {{ rumReport.length }}
+            </div>
+            <table
+              class="w-full divide-y divide-neutral-400 dark:divide-neutral-600 border-b border-neutral-400 dark:border-neutral-600"
+            >
+              <thead>
+                <tr
+                  class="children:pb-xs children:px-md children:font-bold text-left text-xs uppercase"
+                >
+                  <th scope="col">Owner Name</th>
+                  <th scope="col">Owner Email</th>
+                  <th scope="col">Total RUM</th>
+                  <th scope="col">Owner ID</th>
+                </tr>
+              </thead>
+              <tbody
+                class="divide-y divide-neutral-300 dark:divide-neutral-700"
+              >
+                <tr
+                  v-for="entry in rumReport"
+                  :key="entry.ownerPk"
+                  class="children:px-md children:py-sm children:truncate text-sm font-medium text-neutral-800 dark:text-neutral-200"
+                >
+                  <td class="normal-case">
+                    {{ entry.ownerName }}
+                  </td>
+                  <td class="normal-case">
+                    {{ entry.ownerEmail }}
+                  </td>
+                  <td class="normal-case font-mono">
+                    {{ entry.totalRum }}
+                  </td>
+                  <td class="">
+                    <div
+                      class="xl:max-w-[800px] lg:max-w-[60vw] md:max-w-[50vw] sm:max-w-[40vw] max-w-[150px] truncate font-mono text-xs"
+                    >
+                      {{ entry.ownerPk }}
+                    </div>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </template>
+      </div>
+    </template>
+    <template v-else> Feature not Enabled for account </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import {
+  Icon,
+  VormInput,
+  Stack,
+  ErrorMessage,
+  VButton,
+} from "@si/vue-lib/design-system";
+import { useAuthStore } from "@/store/auth.store";
+import { useWorkspacesStore } from "@/store/workspaces.store";
+import { useFeatureFlagsStore } from "@/store/feature_flags.store";
+
+const authStore = useAuthStore();
+const workspacesStore = useWorkspacesStore();
+const featureFlagsStore = useFeatureFlagsStore();
```

Comment:
I might be mis-remembering, but I think we've done a similar motion in the new UI while avoiding using the old pinia stores. I think it was in our work to bring approvals / approval requirements to the new UI? I did a cursory search, but I'd check if we could piggyback off that work to reduce dependence on these stores since they may bring in side effects.